recipe/meta.yaml

```diff
@@ -24,38 +28,35 @@ build:
     - MIGRATING={{ migrating }}
 
 requirements:
-  {% if not win %}
-  build:
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - ipython >=7.23.1                       # [build_platform != target_platform]
-    - jupyter_client >=6.1.12                # [build_platform != target_platform]
-    - packaging                              # [build_platform != target_platform]
-    - psutil                                 # [build_platform != target_platform]
-    - python                                 # [build_platform != target_platform]
-  {% endif %}
   host:
     - debugpy >=1.0.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - packaging
     - pip
     - psutil
-    - python
+    - python 3.9
   run:
-    - appnope  # [osx]
+    {% if noarch_platform == "unix" %}
+    - __unix
+    - appnope
+    {% else %}
+    - __win
+    {% endif %}
     - debugpy >=1.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - matplotlib-inline >=0.1
     - nest-asyncio
     - packaging
     - psutil
-    - python
+    - python >=3.7
```

Comment:
Why is the version constraint needed?

Reply:
(also, jives with https://github.com/ipython/ipykernel/blob/v6.13.0/setup.py#L70)

Reply:
this is the actual runtime python dependency of the package